### PR TITLE
Recipe for Bic (BeInterfaceCreator) (App doesn't build on Haiku)

### PR DESCRIPTION
**Pull request by arnavbhatt288**
This is the recipe for Bic (BeInterfaceCreator).
Keep in mind that the actual application is not building properly on Haiku so I am marking this recipe as broken.
Also, I don't know the name of the creator and the date it was created.

### CODE REVIEW

File: haiku-apps/bic/bic-1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,43 @@
+SUMMARY="WYSIWYG GUI designer"
+DESCRIPTION="BeInterfaceCreator is WYSIWYG GUI designer for Haiku."
+HOMEPAGE="https://github.com/HaikuArchives/BeInterfaceCreator"
+COPYRIGHT="Unknown"
+LICENSE="GPL"
+REVISION="1"
+srcGitRev=""
```
Comment (korli) [inline haiku-apps/bic/bic-1.0.recipe]: Well you need to fill srcGitRev for the recipe to make sense.

File: haiku-apps/bic/bic-1.0.recipe [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+SOURCE_URI="https://github.com/HaikuArchives/BeInterfaceCreator/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256=""
```
Comment (korli) [inline haiku-apps/bic/bic-1.0.recipe]: Same for CHECKSUM_SHA256.

**Comment by korli** [comment]
Please rename to bic-1.0~git.recipe (this way one knows it's not a release).

**Comment by scottmc** [comment]
If you green the source code you will see that the author might be: 
2001 Devulder Jpaul
I just looked through a few files and located that.  Try running green for "(c)" or "copyright" and see what you find.

File: haiku-apps/bic/bic-1.0.recipe (hunk 1 above)
Comment (arnavbhatt288) [reply haiku-apps/bic/bic-1.0.recipe]: I thought that I would fill it later. Oh well

File: haiku-apps/bic/bic-1.0.recipe (hunk 2 above)
Comment (arnavbhatt288) [reply haiku-apps/bic/bic-1.0.recipe]: Alright

**Comment by arnavbhatt288** [comment]
> 2001 Devulder Jpaul

It is Devulder Jpaul

**Comment by arnavbhatt288** [comment]
@kroli Can we do anything about this?

**Pull request merged**